src/App.vue

```diff
@@ -85,58 +73,35 @@
           });
         }
 
-        this.$store.dispatch('setTipsOrdering', this.tipsOrdering);
-
         // filter tips by language from backend
-        // if (langTips) tips = tips.filter(tip => langTips.some(url => tip.url === url));
+        if (langTips) tips = tips.filter(tip => langTips.some(url => tip.url === url));
 
         // add preview to tips from backend
-        if (this.tipsPreview) {
+        if (tipsPreview) {
           tips = tips.map(tip => {
             tip.preview = tipsPreview.find(preview => preview.requestUrl === tip.url);
             return tip;
           });
         }
 
-        this.tempTips = tips;
-        this.$store.commit('UPDATE_TIPS', this.tempTips);
-        this.$store.commit('UPDATE_TOPICS', Object.entries(topics).sort((a, b) => new BigNumber(b[1]).minus(a[1]).toNumber()));
+        // add currency to tips
+        if (rates) {
+          tips = tips.map(tip => {
+            tip.fiatValue = (tip.total_amount * rates.aeternity[this.current.currency]).toFixed(2);
+            return tip;
+          })//.filter(tip => tip.total_amount * (rates.aeternity['usd']).toFixed(2) > 0.01);
+        }
 
         stats.height = await aeternity.client.height();
-        this.$store.commit('UPDATE_STATS', stats);
 
-        this.asyncAddCurrency();
+        this.$store.dispatch('setTipsOrdering', tipOrdering);
```

Comment:
Should we check if these are valid before we push them to the store?

Reply:
I don't think so for now, but we should reconsider if they are needed in the store at all.